manifests/repository.pp

```diff
@@ -42,6 +42,9 @@
 
   if $gpg_key_fingerprint {
     $_gpg_key_fingerprint = $gpg_key_fingerprint
+  } elsif versioncmp($release, '21') >= 0 {
+    # >= bareos 20
+    $_gpg_key_fingerprint = '91DA 1DC3 564A E20A 76C4  CA88 E019 57D6 C9FE D482'
```

Comment:
This should be in a different PR

Reply:
I had to create a new PR to link it to the right brunch: https://github.com/voxpupuli/puppet-bareos/pull/99
Now there should be order in these changes.